fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -34,6 +34,7 @@
       mods:genre = 'Academic theses'.
     -->
     <xsl:apply-templates mode="utk_ir_MODS" select="$content//mods:mods[1]"/>
+	<xsl:apply-templates mode="utk_mods_etd" select="$content//mods:mods[1]"/>
```

Comment:
Let's drop this. We don't need another mode at this point.

Reply:
Done.  Remove mode="utk_mods_etd"  line.

Reply:
Also removed extra template just above my target template in the code.  Brider looked at the file and suggested it..  About 16 lines.